libcontainer/rootfs_linux.go

```diff
@@ -180,18 +180,22 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string) error {
 			if err != nil {
 				return err
 			}
+			relDir, err := filepath.Rel(mm.Root, dir)
```

Comment:
I don't understand. Can you elaborate?
It seems like this `mm.Root` always prepended to cgroup path in `/proc/self/cgroup`
For example if I'm in parent container, I have `mm.Root` like
```
/system.slice/docker-bae3c1be58e13929aa6ae2f10ec958e5e27e9a4fb178f256e229c5bc54212d66.scope
```
and in new container after creation will be in `/proc/self/cgroup` something like:
```
/system.slice/docker-bae3c1be58e13929aa6ae2f10ec958e5e27e9a4fb178f256e229c5bc54212d66.scope/docker/ID
```
I'll retry now though